CHANGELOG.md

```diff
@@ -2,13 +2,177 @@
+is older than 1 major version apart. You can use --skip-version-check flag to
+bypass version check.
+
+Take a look at component compatibility guarantees in the documentation:
+https://goteleport.com/docs/setup/operations/upgrading/#component-compatibility.
+
+#### HTTP_PROXY for reverse tunnels
+
+Reverse tunnel connections will now respect HTTP_PROXY environment variables.
+This may result in reverse tunnel agents not being able to re-establish
+connections if the HTTP proxy is set in their environment and does not allow
+connections to the Teleport proxy.
+
+Refer to the following documentation section for more details:
+https://goteleport.com/docs/setup/reference/networking/#http-connect-proxies.
+
+#### New APT repos
+
+With Teleport 10 we’ve migrated to new APT repositories that now support
+multiple release channels, Teleport versions and OS distributions. The new
+repositories have been backfilled with Teleport versions starting from 6.2.31
+and we recommend upgrading to them. The old repositories will be maintained for
+the foreseeable future.
+
+See updated installation instructions:
+https://goteleport.com/docs/server-access/getting-started/#step-14-install-teleport-on-your-linux-host.
+
+#### Removed “tctl access ls”
+
+The “tctl access ls” command that returned information about user server access
+within the cluster was removed. Please use a previous tctl version if you’d like
+to keep using it.
+
 #### Relaxed session join permissions
 
-In previous versions of Teleport users need full access to the node/Kubernetes pod in order to join a session. With Teleport 10.0 we have relaxed this requirement. Joining sessions remains deny-by-default as of Teleport 9.0 but now only `join_policy` statements as described in the [Moderated Sessions Guide](https://goteleport.com/docs/access-controls/guides/moderated-sessions/) are checked for session join RBAC.
+In previous versions of Teleport users needed full access to a node/Kubernetes
+pod in order to join a session. Teleport 10 relaxes this requirement. Joining
+sessions remains deny-by-default but now only “join_policy” statements are
+checked for session join RBAC.
+
+See moderated sessions guide for more details:
+https://goteleport.com/docs/access-controls/guides/moderated-sessions/.
+
+#### Github connectors
+
+Github connector’s “teams_to_logins” field is deprecated in favor of the new
+“teams_to_roles” field. The old field will be removed in a future release.
+
+#### Teleport FIPS AWS endpoints
+
+Teleport 10 will now automatically use FIPS endpoints for AWS S3 and DynamoDB
+when started with --fips flag. You can use the “use_fips_endpoint=false”
+connection endpoint option to use regular endpoints for Teleport in FIPS mode,
+for example:
+
+s3://bucket/path?region=us-east-1&use_fips_endpoint=false
+
+See S3/DynamoDB backends documentation for more information:
+https://goteleport.com/docs/setup/reference/backends/#s3.
 
 ## 8.0.0
```

Comment:
The changelog appears to be missing releases between 8.0.0 and 10.0.0

Reply:
Yeah this is an artifact of the fact that 9.0 changelogs are only updated in branch/v9. I think we have an issue somewhere to fix this.